edward/inferences/klqp.py

```diff
@@ -53,16 +53,25 @@ class KLqp(VariationalInference):
   def __init__(self, *args, **kwargs):
     super(KLqp, self).__init__(*args, **kwargs)
 
-  def initialize(self, n_samples=1, *args, **kwargs):
+  def initialize(self, n_samples=1, kl_scaling={}, *args, **kwargs):
```

Comment:
instead of the default argument being an empty dict, i recommend using `None`. also see http://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

this is a bit safer. and in the function, you can write
```python
if kl_scaling is None:
  kl_scaling = {}
```